src/js/modules/infragistics.datasource.js

```diff
@@ -4359,8 +4372,8 @@
 				/*A.T. workaround for jQuery's 1.5 and above bug related to dataFilter and success callback.
 				We need to explicitly set the dataType to "text" when manually parsing it */
 				/* get jquery version */
-				if ($.fn.jquery) {
-					ver = $.fn.jquery.split(".");
+				if (jQuery.fn.jquery) {
```

Comment:
@skrustev Didn't we make some changes to always use `$` instead of mix of `$` and `jQuery` because of webpack, or is it the other way around?